Commited in 2.32

commit 5e77ec7c6e64d67f048caa5d4dd81f10fca3dffc
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Feb 9 22:44:08 2020 +0000

    htl: Add support for semaphore maximum value

diff --git a/sysdeps/mach/hurd/bits/local_lim.h b/sysdeps/mach/hurd/bits/local_lim.h
index 348eee74bb..8e781e4965 100644
--- a/sysdeps/mach/hurd/bits/local_lim.h
+++ b/sysdeps/mach/hurd/bits/local_lim.h
@@ -41,3 +41,6 @@
 
 /* The number of threads per process.  */
 #define _POSIX_THREAD_THREADS_MAX	64
+
+/* Maximum value the semaphore can have.  */
+#define SEM_VALUE_MAX   (2147483647)
